ts-boilerplate-esm/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "ts-boilerplate-common",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "tsc && node dist/index.js"
+  },
+  "author": "",
+  "license": "MIT",
+  "description": "",
+  "dependencies": {
+    "express": "^4.21.1"
+  },
+  "devDependencies": {
+    "@types/express": "^5.0.0",
+    "@types/node": "^22.9.1",
+    "ts-node": "^10.9.2",
```

Comment:
Why include `ts-node` here if it's not used in the scripts?
I think there are 2 choices here:
- Either keep the boiletplate as simple as possibl, remove ts-node and keep the basic `start` script as it is
- Or use a more "class" script approach, with different start script for dev and prod, and  build/build:watch script. But in that case I'd suggest `tsx` instead of `ts-node`, it's a more modern replacement with way better ESM support out of the box.